indexed_gzip/indexed_gzip.pyx

```diff
@@ -660,6 +660,85 @@ cdef class IndexedGzipFile:
         """Currently does nothing. """
         pass
 
+    def export_index(self, filename=None, fileobj=None):
+        '''Export index data to the given file. Either ``filename`` or
+        ``fileobj`` should be specified, but not both. ``fileobj`` should be
+        opened in 'wb' mode.
+
+        :arg filename: Name of the file.
+        :arg fileobj:  Open file handle.'''
+
+        if filename is None and fileobj is None:
+            raise ValueError('One of filename or fileobj must be specified')
+
+        if filename is not None and fileobj is not None:
+            raise ValueError(
+                'Only one of filename or fileobj must be specified')
+
+        if filename is not None:
+            fileobj = open(filename, 'wb')
+            close_file = True
+
+        else:
+            close_file = False
+            if fileobj.mode not in 'wb':
```

Comment:
Any reason for using `not in`? This will pass if `fileobj.mode` is any of `'w'`, `'b'` (will never happen), or `'wb'`. I don't know much about Windows, so am not completely sure of the ramifications.

Reply:
I was checking if mode is either 'ab' or 'wb', then I changed my mind. Apparently I forgot to remove "in".